python/lsst/the/monster/isolate_and_transform.py

```diff
@@ -0,0 +1,149 @@
+import os
+from smatch import Matcher
+
+from lsst.pipe.tasks.isolatedStarAssociation import IsolatedStarAssociationTask
+
+from lsst.the.monster.splinecolorterms import ColortermSpline
```

Comment:
This should be `from .splinecolorterms import ColortermSpline` and similar below.